src/OpenLoco/src/Windows/News/News.cpp

```diff
@@ -851,6 +851,7 @@ namespace OpenLoco::Ui::Windows::NewsWindow
 
         static constexpr WindowEventList kEvents = {
             .onMouseUp = onMouseUp,
+            .onResize = initViewport,
```

Comment:
I would somewhat prefer if we have an explicit functions for onResize and viewportRotate, this feels prone to accidentally do something it shouldn't.

Reply:
I would prefer that, too. However, looking at `initViewport` in its current state, I believe it's overdue a refactor, anyway. I would prefer for it to be split up into several functions, and made into different events then.

Reply:
I see, better left for another PR then.